From ca83683b9e8072a6f2774415f7fd3d22c5cb9a3a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Wed, 24 Aug 2016 11:13:03 +0200
Subject: [PATCH 5/9] include: SwitchToThread is always available on Win10

---
 mingw-w64-headers/include/processthreadsapi.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mingw-w64-headers/include/processthreadsapi.h b/mingw-w64-headers/include/processthreadsapi.h
index 7e14eb9..c4629f7 100755
--- a/mingw-w64-headers/include/processthreadsapi.h
+++ b/mingw-w64-headers/include/processthreadsapi.h
@@ -167,6 +167,7 @@ WINBASEAPI WINBOOL WINAPI TerminateProcess (HANDLE hProcess, UINT uExitCode);
   WINBASEAPI WINBOOL WINAPI GetExitCodeThread (HANDLE hThread, LPDWORD lpExitCode);
 #if _WIN32_WINNT >= 0x0A00
   WINBASEAPI DWORD WINAPI QueueUserAPC (PAPCFUNC pfnAPC, HANDLE hThread, ULONG_PTR dwData);
+  WINBASEAPI WINBOOL WINAPI SwitchToThread (VOID);
 #endif
   WINBASEAPI DWORD WINAPI SuspendThread (HANDLE hThread);
   WINBASEAPI DWORD WINAPI ResumeThread (HANDLE hThread);
-- 
2.9.3

