From 46ac62d629a2aad4fad43b68f344b0b3777dadd1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Wed, 9 Nov 2016 10:28:24 +0100
Subject: [PATCH 8/9] dpapi: Crypt*Data functions are allowed on Win10

---
 mingw-w64-headers/include/dpapi.h | 24 +++++++++++++++---------
 1 file changed, 15 insertions(+), 9 deletions(-)

diff --git a/mingw-w64-headers/include/dpapi.h b/mingw-w64-headers/include/dpapi.h
index 1e685ea..8ff8630 100644
--- a/mingw-w64-headers/include/dpapi.h
+++ b/mingw-w64-headers/include/dpapi.h
@@ -14,7 +14,7 @@
 extern "C" {
 #endif
 
-#if WINAPI_FAMILY_PARTITION (WINAPI_PARTITION_DESKTOP)
+#if WINAPI_FAMILY_PARTITION (WINAPI_PARTITION_DESKTOP) || _WIN32_WINNT >= _WIN32_WINNT_WIN10
 
 #define CRYPTPROTECT_DEFAULT_PROVIDER { 0xdf9d8cd0, 0x1501, 0x11d1,{ 0x8c, 0x7a, 0x00, 0xc0, 0x4f, 0xc2, 0x97, 0xeb } }
 
@@ -41,12 +41,6 @@ extern "C" {
 #define CRYPTPROTECT_FIRST_RESERVED_FLAGVAL 0x0fffffff
 #define CRYPTPROTECT_LAST_RESERVED_FLAGVAL 0xffffffff
 
-#define CRYPTPROTECTMEMORY_BLOCK_SIZE 16
-
-#define CRYPTPROTECTMEMORY_SAME_PROCESS 0x0
-#define CRYPTPROTECTMEMORY_CROSS_PROCESS 0x1
-#define CRYPTPROTECTMEMORY_SAME_LOGON 0x2
-
 #ifndef CRYPTO_BLOBS_DEFINED
 #define CRYPTO_BLOBS_DEFINED
   typedef struct _CRYPTOAPI_BLOB {
@@ -77,8 +71,7 @@ extern "C" {
 
   DPAPI_IMP WINBOOL WINAPI CryptProtectData (DATA_BLOB *pDataIn, LPCWSTR szDataDescr, DATA_BLOB *pOptionalEntropy, PVOID pvReserved, CRYPTPROTECT_PROMPTSTRUCT *pPromptStruct, DWORD dwFlags, DATA_BLOB *pDataOut);
   DPAPI_IMP WINBOOL WINAPI CryptUnprotectData (DATA_BLOB *pDataIn, LPWSTR *ppszDataDescr, DATA_BLOB *pOptionalEntropy, PVOID pvReserved, CRYPTPROTECT_PROMPTSTRUCT *pPromptStruct, DWORD dwFlags, DATA_BLOB *pDataOut);
-  DPAPI_IMP WINBOOL WINAPI CryptProtectMemory (LPVOID pDataIn, DWORD cbDataIn, DWORD dwFlags);
-  DPAPI_IMP WINBOOL WINAPI CryptUnprotectMemory (LPVOID pDataIn, DWORD cbDataIn, DWORD dwFlags);
+
 #if NTDDI_VERSION >= 0x06020000
   WINBOOL WINAPI CryptProtectDataNoUI (DATA_BLOB *pDataIn, LPCWSTR szDataDescr, DATA_BLOB *pOptionalEntropy, PVOID pvReserved, CRYPTPROTECT_PROMPTSTRUCT *pPromptStruct, DWORD dwFlags, const BYTE *pbOptionalPassword, DWORD cbOptionalPassword, DATA_BLOB *pDataOut);
   WINBOOL WINAPI CryptUnprotectDataNoUI (DATA_BLOB *pDataIn, LPWSTR *ppszDataDescr, DATA_BLOB *pOptionalEntropy, PVOID pvReserved, CRYPTPROTECT_PROMPTSTRUCT *pPromptStruct, DWORD dwFlags, const BYTE *pbOptionalPassword, DWORD cbOptionalPassword, DATA_BLOB *pDataOut);
@@ -89,6 +82,19 @@ extern "C" {
 
 #endif
 
+#if WINAPI_FAMILY_PARTITION (WINAPI_PARTITION_DESKTOP)
+
+#define CRYPTPROTECTMEMORY_BLOCK_SIZE 16
+
+#define CRYPTPROTECTMEMORY_SAME_PROCESS 0x0
+#define CRYPTPROTECTMEMORY_CROSS_PROCESS 0x1
+#define CRYPTPROTECTMEMORY_SAME_LOGON 0x2
+
+DPAPI_IMP WINBOOL WINAPI CryptProtectMemory (LPVOID pDataIn, DWORD cbDataIn, DWORD dwFlags);
+DPAPI_IMP WINBOOL WINAPI CryptUnprotectMemory (LPVOID pDataIn, DWORD cbDataIn, DWORD dwFlags);
+
+#endif
+
 #ifdef __cplusplus
 }
 #endif
-- 
2.9.3

