From 76fdf024eb822e0cb92e13fca7d633d229c2c548 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Tue, 27 Jun 2017 17:30:24 +0200
Subject: [PATCH] winbase.h: WaitForMultipleObjects is available on UWP

---
 mingw-w64-headers/include/winbase.h | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/mingw-w64-headers/include/winbase.h b/mingw-w64-headers/include/winbase.h
index 74bc512c..be050cc6 100644
--- a/mingw-w64-headers/include/winbase.h
+++ b/mingw-w64-headers/include/winbase.h
@@ -1265,6 +1265,10 @@ extern "C" {
 #define CRITICAL_SECTION_NO_DEBUG_INFO RTL_CRITICAL_SECTION_FLAG_NO_DEBUG_INFO
 #endif
 
+#if WINAPI_FAMILY_PARTITION (WINAPI_PARTITION_DESKTOP) || _WIN32_WINNT >= _WIN32_WINNT_WIN10
+  WINBASEAPI DWORD WINAPI WaitForMultipleObjects (DWORD nCount, CONST HANDLE *lpHandles, WINBOOL bWaitAll, DWORD dwMilliseconds);
+#endif
+
 #if WINAPI_FAMILY_PARTITION (WINAPI_PARTITION_DESKTOP)
   typedef enum _DEP_SYSTEM_POLICY_TYPE {
     DEPPolicyAlwaysOff = 0,
@@ -1286,7 +1290,6 @@ extern "C" {
 #define SET_TAPE_DRIVE_INFORMATION 1
 
   WINBASEAPI WINBOOL WINAPI PulseEvent (HANDLE hEvent);
-  WINBASEAPI DWORD WINAPI WaitForMultipleObjects (DWORD nCount, CONST HANDLE *lpHandles, WINBOOL bWaitAll, DWORD dwMilliseconds);
   WINBASEAPI ATOM WINAPI GlobalDeleteAtom (ATOM nAtom);
   WINBASEAPI WINBOOL WINAPI InitAtomTable (DWORD nSize);
   WINBASEAPI ATOM WINAPI DeleteAtom (ATOM nAtom);
-- 
2.11.0

