--- configure.ac.old	2018-02-16 14:27:06.664541960 +0100
+++ configure.ac	2018-02-16 14:27:10.608542087 +0100
@@ -31,7 +31,7 @@
 m4_include([config/isl.m4])
 
 AC_INIT(move-if-change)
-AC_PREREQ(2.64)
+AC_PREREQ(2.69)
 AC_DISABLE_OPTION_CHECKING
 
 progname=$0
--- config/override.m4.old	2018-02-16 17:00:09.972292042 +0100
+++ config/override.m4	2018-02-16 17:00:13.912292169 +0100
@@ -29,7 +29,7 @@
 
 dnl Ensure exactly this Autoconf version is used
 m4_ifndef([_GCC_AUTOCONF_VERSION],
-  [m4_define([_GCC_AUTOCONF_VERSION], [2.64])])
+  [m4_define([_GCC_AUTOCONF_VERSION], [2.69])])
 
 dnl Test for the exact version when AC_INIT is expanded.
 dnl This allows to update the tree in steps (for testing)
