From 859fc95e5e8248a986d4f0327f36bb2c46dcc86c Mon Sep 17 00:00:00 2001
From: Bastien Penavayre <bastien@videolabs.io>
Date: Thu, 2 Jun 2016 17:18:15 +0200
Subject: [PATCH] Disable arm testing for zlib on x86 systems

---
 ports/zlib/build.sh | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/ports/zlib/build.sh b/ports/zlib/build.sh
index a2e6f77..2cc44b2 100644
--- a/ports/zlib/build.sh
+++ b/ports/zlib/build.sh
@@ -58,9 +58,5 @@ TestStep() {
     # Run the minigzip tests again but with x86-32 and arm translations
     WriteLauncherScript minigzip minigzip.x86-32.nexe
     RunMinigzip
-    if [[ ${SEL_LDR_SUPPORTS_ARM} == 1 ]]; then
-      WriteLauncherScript minigzip minigzip.arm.nexe
-      RunMinigzip
-    fi
   fi
 }
-- 
2.7.4

